builtin/game/chatcommands.lua

```diff
@@ -82,7 +82,8 @@ core.register_chatcommand("me", {
 core.register_chatcommand("admin", {
 	description = "Show the name of the server owner",
 	func = function(name)
-		local admin = minetest.settings:get("name")
+		local admin =  minetest.settings:get("admin_name") ~= "" and minetest.settings:get("admin_name")
```

Comment:
extra space

Reply:
\* core.settings
For these two lines

Reply:
I like the shortening...

Reply:
Alas, the shortening would return true if amin_name is not empty (not the value)